packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -6,7 +6,13 @@ import { Autocomplete, Button, DesktopWrapper, Input, MobileWrapper, Text, Selec
 import { Formik, Field } from 'formik';
 import { localize, Localize } from '@deriv/translations';
 import { formatInput, WS } from '@deriv/shared';
-import { isSequentialNumber, isRecurringNumberRegex, getDocumentData, getRegex } from './utils';
+import {
+    isSequentialNumber,
+    isRecurringNumberRegex,
+    getDocumentData,
+    getRegex,
+    preventEmptyClipboardPast,
```

Comment:
Is that a typo? what is "past"?
```suggestion
    preventEmptyClipboardPast,
```

Reply:
yes, sure. it is typo.. omg.. thanks a lot